functors_task_1/student/include/impl.h

```diff
@@ -40,9 +66,31 @@ struct MixingPolicy
      * Whiskey + SevenUp = SevenPlusSeven;
      * Others + Others = Oops;
      */
-    static Cocktail mix(/**???*/)
+    static Cocktail mix(AlcoholDrink alchohol, NonAlcoholDrink nonAlcohol)
+    {
+        std::unordered_map<AlcoholDrink, std::unordered_map<NonAlcoholDrink, Cocktail>> cocktailes =
```

Comment:
1. Yep, mapping is a good pattern and should be used here, but please, implement this task without containers usage. You can easily replace it with a switch.
2. Why not const?
3. Why do you recreate this map on every function call?
4. You can remove "="